src/GitHub.TeamFoundation.14/Sync/GitHubPublishSection.cs

```diff
@@ -114,11 +117,55 @@ public void ShowPublish()
                 // there's no real cancel button in the publish form, but if support a back button there, then we want to hide the form
                 IsVisible = false;
                 if (success)
+                {
                     ServiceProvider.TryGetService<ITeamExplorer>()?.NavigateToPage(new Guid(TeamExplorerPageIds.Home), null);
+                    HandleCreatedRepo(ActiveRepo);
+                }
             });
             uiProvider.Run(controller);
         }
 
+        void HandleCreatedRepo(ILocalRepositoryModel newrepo)
+        {
+            var msg = string.Format(CultureInfo.CurrentUICulture, Constants.Notification_RepoCreated, newrepo.Name, newrepo.CloneUrl);
+            msg += " " + string.Format(CultureInfo.CurrentUICulture, Constants.Notification_CreateNewProject, newrepo.LocalPath);
+            ShowNotification(newrepo, msg);
+        }
+
+        private void ShowNotification(ILocalRepositoryModel newrepo, string msg)
+        {
+            var teServices = ServiceProvider.TryGetService<ITeamExplorerServices>();
+
+            teServices.ClearNotifications();
+            teServices.ShowMessage(
+                msg,
+                    new RelayCommand(o =>
```

Comment:
Ah it's the way it's formatted in the [ConnectSectionClass](https://github.com/github/VisualStudio/blob/a140ebdb69407ed1df907f1c6de48ee46350f149/src/GitHub.TeamFoundation.14/Connect/GitHubConnectSection.cs#L288-L289) where I <s>copied</s> replicated the notification behavior. But can fix it there too if it's better readability.